test error: test_meta_tensor

**Describe the bug**
```
01:00:56  ======================================================================
01:00:56  FAIL: test_pickling (tests.test_meta_tensor.TestMetaTensor)
01:00:56  ----------------------------------------------------------------------
01:00:56  Traceback (most recent call last):
01:00:56    File "/home/jenkins/agent/workspace/Monai-latest-image/tests/test_meta_tensor.py", line 252, in test_pickling
01:00:56      self.check(m2, m, ids=False)
01:00:56    File "/home/jenkins/agent/workspace/Monai-latest-image/tests/test_meta_tensor.py", line 81, in check
01:00:56      self.assertIsInstance(out, type(orig))
01:00:56  AssertionError: tensor([[[0.3962, 0.9173, 0.7437, 0.3341, 0.4098, 0.3432, 0.1271, 0.6718],
01:00:56           [0.3670, 0.5618, 0.4506, 0.5650, 0.2895, 0.9468, 0.6888, 0.6819],
01:00:56           [0.3909, 0.7587, 0.9295, 0.4800, 0.9351, 0.0733, 0.4742, 0.0840],
01:00:56           [0.5425, 0.4293, 0.2237, 0.9117, 0.3608, 0.4339, 0.7344, 0.9005],
01:00:56           [0.5106, 0.0122, 0.6607, 0.0010, 0.6300, 0.2717, 0.6291, 0.3825],
01:00:56           [0.0115, 0.9764, 0.6277, 0.1557, 0.6526, 0.6177, 0.0266, 0.6330],
01:00:56           [0.7035, 0.0930, 0.4120, 0.8611, 0.0737, 0.5855, 0.8366, 0.0864],
01:00:56           [0.5331, 0.4385, 0.8556, 0.1940, 0.2305, 0.7927, 0.0265, 0.0143],
01:00:56           [0.1303, 0.9910, 0.4199, 0.3048, 0.2329, 0.5578, 0.5873, 0.6963],
01:00:56           [0.1361, 0.1508, 0.0750, 0.4933, 0.5001, 0.1407, 0.2277, 0.2387]]]) is not an instance of <class 'monai.data.meta_tensor.MetaTensor'>
01:00:56  
01:00:56  ----------------------------------------------------------------------
```

from container 21.02 Pytorch version: 1.8.0a0+52ea372

--- a/tests/test_meta_tensor.py
+++ b/tests/test_meta_tensor.py
@@ -24,10 +24,8 @@ from parameterized import parameterized
 from monai.data.meta_obj import get_track_meta, get_track_transforms, set_track_meta, set_track_transforms
 from monai.data.meta_tensor import MetaTensor
 from monai.utils.enums import PostFix
-from monai.utils.module import get_torch_version_tuple
+from monai.utils.module import pytorch_after
 from tests.utils import TEST_DEVICES, assert_allclose, skip_if_no_cuda
-
-PT_VER_MAJ, PT_VER_MIN = get_torch_version_tuple()
 
 DTYPES = [[torch.float32], [torch.float64], [torch.float16], [torch.int64], [torch.int32]]
 TESTS = []
@@ -230,7 +228,7 @@ class TestMetaTensor(unittest.TestCase):
             traced_fn = torch.jit.load(fname)
             out = traced_fn(im)
             self.assertIsInstance(out, torch.Tensor)
-            if not isinstance(out, MetaTensor) and PT_VER_MAJ == 1 and PT_VER_MIN <= 9:
+            if not isinstance(out, MetaTensor) and not pytorch_after(1, 9, 1):
                 warnings.warn(
                     "When calling `nn.Module(MetaTensor) on a module traced with "
                     "`torch.jit.trace`, your version of pytorch returns a "
@@ -246,7 +244,7 @@ class TestMetaTensor(unittest.TestCase):
             fname = os.path.join(tmp_dir, "im.pt")
             torch.save(m, fname)
             m2 = torch.load(fname)
-            if not isinstance(m2, MetaTensor) and PT_VER_MAJ == 1 and PT_VER_MIN <= 7:
+            if not isinstance(m2, MetaTensor) and not pytorch_after(1, 8, 1):
                 warnings.warn("Old version of pytorch. pickling converts `MetaTensor` to `torch.Tensor`.")
                 m = m.as_tensor()
             self.check(m2, m, ids=False)

